src/pages/Home/components/ShelterListView/ShelterListView.tsx

```diff
@@ -36,7 +36,10 @@ const ShelterListView = React.forwardRef<HTMLDivElement, IShelterListViewProps>(
     return (
       <div className={cn(className, 'flex flex-col gap-2')}>
         <h1 className="text-[#2f2f2f] font-semibold text-2xl">
-          Abrigos disponíveis ({count})
+          {searchParams.toString()
+            ? `Abrigos encontrados (${count})`
```

Comment:
Não entendi muito bem essa condicional aqui, em teoria ela sempre vai ser [object Object] caso seja um objeto inicializado, e se for null vai dar `Cannot read properties of null`. 

E no total de abrigos faltou o count.

Reply:
Oi @rodrigooler :
Referente a condicional: essa condicional é usada em outra parte para exibir o botão "Limpar filtros". Tendo em vista que esse botão só é exibido após os filtros serem aplicados eu apenas reaproveitei a lógica.
Veja:

> {searchParams.toString() && (
  <Button
    variant="ghost"
    size="sm"
    className="flex gap-2 items-center text-blue-500 hover:text-blue-600 active:text-blue-700"
    onClick={onClearSearch}
    <CircleAlert className="h-5 w-5 stroke-blue-500" />
    Limpar Filtros
  </Button> )}
  >
    

Em cima dessa lógica eu pensei em usar para trocar o titulo de acordo com a aplicação do filtro ou não. Já que existe algo que confirma a aplicação dos filtros, eu precisava apenas usar.
Enviarei a alteração acrescentado o {count} em ambas as situações, com filtro aplicado e não.

> <h1 className="text-[#2f2f2f] font-semibold text-2xl">
  {searchParams.toString()
    ? `Abrigos encontrados (${count})`
    : `Total de abrigos  (${count})`
  }
</h1>

A lógica por trás dessa condicional é verificar se há algum parâmetro de pesquisa presente. Se houver, a string resultante será uma string não vazia, que é considerada um valor verdadeiro em JavaScript.

Reply:
@rodrigooler , eu fiz as alterações e atualizei o PR. Por favor, dê uma olhada quando tiver um momento. Obrigado!

Reply:
oi @Tiago-Silva, tudo bom?, pelo que entendi, a issue pede mostrar apenas a quantidade de abrigos disponíveis, independente se estamos passando query params na url da página. 

No caso do botão que vc citou faz sentido pq só se deve limpar uma busca de filtro de algo está sendo buscado, ou seja, passado como query param e recuperado através do hook `useSearchParams`. 

No caso dessa task nós não precisamos olhar para as query params, e sim para a prop `count` passada. Observe que o componente pai (`<Home`) está passando a prop `count` para `<ShelterListView` aqui https://github.com/Lmedeiros-leiman/frontend/blob/2f807b7fc0bd3e6403f5484cc7397974b0389760/src/pages/Home/Home.tsx#L119

Podemos resolver isso no front da seguinte maneira: 
- ao invés de passar `shelters.count` na linha 119, podemos filtrar os shelters com `capacity > shelteredPeople` e passar o length do resultado

Reply:
Bom dia @larissapissurno Obrigado pelo feedback.
Eu não entendi muito bem a sua idéia porque, a proposta para solução:
![Captura de tela de 2024-05-24 07-07-26](https://github.com/SOS-RS/frontend/assets/9256219/23136caf-1f19-400b-9b08-a64bb18d9adc)
Baseado nessa proposta o objetivo é mudar o titulo caso a lista de abrigos que estão sendo exibidas seja proveniente da pesquisa com filtros e ter um titúlo padrão para a pesquisa default.
Usar o {count} para isso não parece a priori uma boa idéia, porque essa varíavel em teoria sempre retornará maior que zero uma vez que a aplicação estiver em uso (ter pelo menos um abrigo cadastrado), porque ao carregar a página já é feito uma pesquisa de todos os abrigos cadastrados.
Então pela proposta sugerida a melhor opção foi olhar para validar a aplicação dos filtros ou não, mas, de qualquer forma uma lista de abrigos será exibida.
Quando você pontou para usar a capacidade de pessoas por abrigo:
```typescript
 shelters.count  //ao invés de passar na linha 119, podemos filtrar os shelters com 
capacity > shelteredPeople // e passar o length do resultado
``` 
Ainda assim ficou consufo para mim, porque se houver pelo menos um abrigo cadastrado, em teoria sua capacidade será maior que zero e como mencionei ao carregar a página já é realizado uma consulta default retornando uma lista de abrigo.

@larissapissurno estou a disposição.

Reply:
boa noite @Tiago-Silva, peço desculpas, eu li rapidamente a issue e imaginei que a solução fosse para apenas mostrar a quantidade de abrigos disponíveis, sem considerar o filtro. 
Vou revisar novamente seu PR levando em conta a solução proposta na issue 🙏